driver/web/rest/adminapis.go

```diff
@@ -56,6 +56,25 @@ func (h AdminApisHandler) GetCovid19Config(current model.User, group string, w h
 	w.Write(data)
 }
 
+func (h AdminApisHandler) GetCovid19Configs(current model.User, group string, w http.ResponseWriter, r *http.Request) {
```

Comment:
When we add a new API you have to add a documentation for it. The documentation is here - https://api-dev.rokwire.illinois.edu/health/doc/ui/

We add the documentation in a descriptive way like this https://github.com/rokwire/health-building-block/blob/3c538d37004057ee1b77bedeb4c66433b6ad0dd5/driver/web/rest/adminapis.go#L86
Once it is described as comments then we use the swagger tool for generating it. The command is "make swagger"